pandas/_libs/tslib.pyx

```diff
@@ -100,16 +100,16 @@ iNaT = NPY_NAT
 
 
 from tslibs.timezones cimport (
-    is_utc, is_tzlocal, _is_fixed_offset,
+    is_utc, is_tzlocal, is_fixed_offset,
     treat_tz_as_dateutil, treat_tz_as_pytz,
     get_timezone, get_utcoffset, maybe_get_tz,
-    _get_dst_info
+    get_dst_info
     )
 from tslibs.timezones import (  # noqa
     get_timezone, get_utcoffset, maybe_get_tz,
-    _p_tz_cache_key, dst_cache,
-    _unbox_utcoffsets,
-    _dateutil_gettz
+    p_tz_cache_key, dst_cache,
```

Comment:
are all of these actually used in tslib or just other modules import them? if so then just import directly from time zones